packages/aws-client/src/SQS.ts

```diff
@@ -139,16 +143,25 @@ export const receiveSQSMessages = async (
   return <SQSMessage[]>(messages.Messages ?? []);
 };
 
+/**
+ * Extract SQS message body
+ *
+ * @param {SQSRecord | AWS.SQS.Message} message - SQS message
+ * @param {string} message.body - optional key for message body
+ * @param {string} message.Body - optional key for message body
+ * @returns {StepFunctionEventBridgeEvent | CumulusMessage} an AWS SQS message body
```

Comment:
This annotation appears to be incorrect (see other comment in the PR re: annotations/.ts as well)